work-order-service/src/main/java/org/mkgroup/zaga/workorderservice/service/WorkOrderService.java

```diff
@@ -188,33 +183,16 @@ public SAPResponse addWorkOrder(WorkOrderDTO workOrderDTO) throws Exception {
 			}
 			WorkOrder wo = getOneW(workOrderId);
 			//System.out.println(wo.getWorkers().size()+"AAA");
+			Log.info("Work order creation successfuly finished");
```

Comment:
We don't use com.sun.tools.sjavac.Log rather Logger log.

Reply:
Resolved.